generator/explores/explore.py

```diff
@@ -15,6 +15,7 @@ class Explore:
     """A generic explore."""
 
     name: str
+    namespace: str
```

Comment:
I'm not seeing where this field is being used. Can you point it out to me?

Reply:
This actually turned out to be unnecessary, I removed it.